sklearn/neighbors/_binary_tree.pxi

```diff
@@ -979,6 +978,17 @@ cdef class BinaryTree:
             self.node_bounds.base,
         )
 
+    def get_valid_metrics(self):
+        """Get list of valid distance metrics.
+
+        Returns
+        -------
+        valid_metrics: list of str
+            List of valid metrics.
+        """
+        return self.valid_metrics
```

Comment:
We might want to use a property instead.

Reply:
```python
@property
def get_valid_metrics(self):
        """Get list of valid distance metrics.
        Returns
        -------
        valid_metrics: list of str
            List of valid metrics.
        """
        return self.valid_metrics
```

Something like this?

Reply:
If it is a property then `valid_metrics` as a name is sufficient.